rspec-mode.el

```diff
@@ -761,7 +761,24 @@ file if it exists, or sensible defaults otherwise."
 
 (defun rspec--docker-default-wrapper (docker-command docker-container command)
   "Function for wrapping a command for execution inside a dockerized environment. "
-  (format "%s %s sh -c \"%s\"" docker-command docker-container command))
+    (if-let ((docker-cwd . ((rspec--docker-find-cwd))))
+        (format "cd %s; %s %s sh -c \"%s\""
```

Comment:
Instead of adding `cd ...`, have you tried adding a binding for `default-directory`? Then you won't need two branches. Just a smaller one for which value of `default-directory` to keep (depending on whether docker-cwd is non-nil).